apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/config/BizConfig.java

```diff
@@ -240,6 +240,11 @@ public boolean isConfigServiceCacheKeyIgnoreCase() {
     return getBooleanProperty("config-service.cache.key.ignore-case", false);
   }
 
+  public boolean isConfigServiceChangeCacheEnabled() {
+    return getBooleanProperty("config-service.change.cache.enabled", false);
```

Comment:
From a logical standpoint, enabling incremental configuration and caching in the config service are independent features. Should we instead add a feature flag named "config-service.incremental.change.enabled"?

Reply:
ok